apm-sniffer/apm-sdk-plugin/kafka-plugin/src/main/java/org/apache/skywalking/apm/plugin/kafka/define/KafkaConsumerInstrumentation.java

```diff
@@ -63,103 +63,103 @@ public class KafkaConsumerInstrumentation extends AbstractKafkaInstrumentation {
     @Override
     public ConstructorInterceptPoint[] getConstructorsInterceptPoints() {
         return new ConstructorInterceptPoint[] {
-            new ConstructorInterceptPoint() {
-                @Override
-                public ElementMatcher<MethodDescription> getConstructorMatcher() {
-                    return takesArgumentWithType(0, CONSTRUCTOR_INTERCEPT_TYPE);
-                }
+                new ConstructorInterceptPoint() {
+                    @Override
+                    public ElementMatcher<MethodDescription> getConstructorMatcher() {
+                        return takesArgumentWithType(0, CONSTRUCTOR_INTERCEPT_TYPE);
+                    }
 
-                @Override
-                public String getConstructorInterceptor() {
-                    return CONSUMER_CONFIG_CONSTRUCTOR_INTERCEPTOR_CLASS;
-                }
-            },
-              new ConstructorInterceptPoint() {
-                  @Override
-                  public ElementMatcher<MethodDescription> getConstructorMatcher() {
-                      return takesArgumentWithType(0, CONSTRUCTOR_INTERCEPT_MAP_TYPE);
-                  }
-
-                @Override
-                public String getConstructorInterceptor() {
-                    return MAP_CONSTRUCTOR_INTERCEPTOR_CLASS;
-                }
-            },
+                    @Override
+                    public String getConstructorInterceptor() {
+                        return CONSUMER_CONFIG_CONSTRUCTOR_INTERCEPTOR_CLASS;
+                    }
+                },
+                new ConstructorInterceptPoint() {
+                    @Override
+                    public ElementMatcher<MethodDescription> getConstructorMatcher() {
+                        return takesArgumentWithType(0, CONSTRUCTOR_INTERCEPT_MAP_TYPE);
+                    }
+
+                    @Override
+                    public String getConstructorInterceptor() {
+                        return MAP_CONSTRUCTOR_INTERCEPTOR_CLASS;
+                    }
+                },
```

Comment:
Please revert re-format.

Reply:
> Please revert re-format.

I have already recerted it, this class dosen't change, I have already rollback it to the original version

Reply:
> I have already recerted it, this class dosen't change, I have already rollback it to the original version

No, you didn't. Change is still there.

Reply:
> > I have already recerted it, this class dosen't change, I have already rollback it to the original version
> 
> No, you didn't. Change is still there.

Those changes are compare to the changes which are not correct I commit before...

Reply:
> > I have already recerted it, this class dosen't change, I have already rollback it to the original version
> 
> No, you didn't. Change is still there.

I dont't know why.... I check out the original branch and copy the codes into the class file, but the IDE says that there is no difference.....